README.md

```diff
@@ -18,13 +23,30 @@ Runs looking glass using the specified config and modules path.
 glass run -c /path/to/config.yaml -m /path/to/modules
 ```
 
-| Command               | Env        | Description                                                                  |
-|-----------------------|------------|------------------------------------------------------------------------------|
-|`--secrets`/`-s` FILE  | SECRETS    | The path to the secrets file.                                                |
-| `--config`/`-c` FILE  | CONFIG     | The path to the configuration file.                                          |
-| `--modules`/`-m` PATH | MODULES    | The path to the modules. Modules will be located under `src` in this folder. |
-| `--log.format` FORMAT | LOG_FORMAT | Specify the format of logs. Supported formats: 'logfmt', 'json'              |
-| `--log.level` LEVEL   | LOG_LEVEL  | Specify the log level. E.g. 'debug', 'warning'. (default: "info")            |
+#### Options
+
+**--secrets** FILE, **-s** FILE, **$SECRETS** *(Optional)*
+
+The path to the YAML secrets file. Secrets can be accessed in the 
+configuration using [Go template syntax](https://golang.org/pkg/text/template/) using the ".Secrets" prefix.
+
+**--config** FILE, **-c** FILE, **$CONFIG** *(Required)*
+
+The path to the YAML configuration file. This file will be parsed
+using [Go template syntax](https://golang.org/pkg/text/template/). 
+
+**--modules** PATH, **-m** PATH, **$MODULES** *(Required)*
+
+The path to the modules. Module must be located under a `src` folder in the modules path.
+The modules path should be writable to `looking-glass`. 
+
+**--log.format** FORMAT, **$LOG_FORMAT** *(Default: "logfmt")*
+
+Specify the format of logs. Supported formats: 'logfmt', 'json'.
+
+**--log.level** LEVEL, **$LOG_LEVEL** *(Default: "info")*
+
+Specify the log level. e.g. 'debug', 'warning'.
```

Comment:
I'd suggest having an exhaustive list of log levels here since depending on the implementations, `trace` and `fatal` are sometimes part of the levels and sometimes not. WDYT?